api/openapi.yaml

```
@@ -986,9 +1000,19 @@ paths:
           $ref: "#/components/responses/UnexpectedProblemResponse"
     post:
       operationId: createGrant
-      summary: Create grant
+      summary: Create a grant
       description: |
-        Create a grant for an entitlement.
+        Grants define a behavior of granting usage for a metered entitlement. They can have complicated recurrence and rollover rules, thanks to which you can define a wide range of access patterns with a single grant, in most cases you don't have to periodically create new grants. You can only issue grants for active metered entitlements.
+
+        A grant defines a given amount of usage that can be consumed for the entitlement. It has an effectiveAt time and a necessary expiration.
+
+        Grants have a priority setting which determines the order in which they're used. Lower numbers have higher priority with 0 being the highest priority.
```

```suggestion
        Grants have a priority setting that determines their order of use. Lower numbers have higher priority, with 0 being the highest priority. 
```

Maybe add an example here